SUMMARY.md

```diff
@@ -136,11 +132,17 @@
   * [Self Service](operations/business-operations/self-service.md)
   * [Salesforce Administration](operations/business-operations/salesforce-administration.md)
 * [Finance](operations/finance/README.md)
-* [Spending Company Money](operations/finance/spending-company-money/README.md)
-  * [Budget](operations/finance/spending-company-money/budget.md)
-  * [Procurement](operations/finance/spending-company-money/procurement.md)
-   * [Purchase Submission Request](/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract.md)
-  * [How to spend company money](company/how-to-guides-for-staff/how-to-spend-company-money/README.md)
+  * [Spending Company Money](operations/finance/spending-company-money/README.md)
+    * [Budget](operations/finance/spending-company-money/budget.md)
+    * [Procurement](operations/finance/spending-company-money/procurement.md)
+      * [Purchase Submission Request](/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract.md)
+  * [Arrange a Bounty Program]
```

Comment:
missing a link?

Reply:
Yeah, I haven't created all the pages which isn't a great user experience. I'll add all the pages first.